concepticondata/conceptlists/Tryon-1983-324.tsv

```diff
@@ -176,7 +176,7 @@ Tryon-1983-324-174	174	my	Possessives	2090	MY
 Tryon-1983-324-175	175	your sg.	Possessives	732	YOUR (SINGULAR)
 Tryon-1983-324-176	176	his	Possessives	2092	HIS OR HER
```

Comment:
Just realized that HIS OR HER is probably worth to be split into:

* HIS OR HER
* HIS
* HER

given that we have only his here.